torch/onnx/_internal/exporter.py

```diff
@@ -522,12 +451,38 @@ def export(self) -> ExportOutput:
             *self.model_args, **self.model_kwargs
         )
 
-        # Export FX graph to ONNX ModelProto.
-        #
-        # Note that ALL kwargs are folded into constants in graph_module, so we don't pass kwargs
-        # to _export.
-        return self.options.fx_tracer._export_fx_to_onnx(
-            self.options, graph_module, updated_model_args
+        # TODO: Design the passes API
+        graph_module = pre_export_passes(self.options, graph_module, updated_model_args)
+
+        from onnxscript.function_libs.torch_lib import (  # type: ignore[import]
+            graph_building as onnxscript_graph_building,
+        )
+
+        onnxscript_graph = onnxscript_graph_building.TorchScriptGraph()
+
+        from torch.onnx._internal.fx import fx_onnx_interpreter
+
+        fx_interpreter = fx_onnx_interpreter.FxOnnxInterpreter(
+            diagnostic_context=self.options.diagnostic_context
+        )
+        fx_interpreter.run(
+            torchscript_tracer=onnxscript_graph_building.TorchScriptTracingEvaluator(
+                onnxscript_graph
+            ),
+            onnxscript_graph=onnxscript_graph,
+            fx_name_to_onnxscript_value={},
```

Comment:
remove `fx_name_to_onnxscript_value` from `fx_interpreter.run`'s argument list and defines inside `fx_interpreter.run`.